docs/library/uos.rst

```diff
@@ -294,5 +302,7 @@ that the block device supports the extended interface.
             (*arg* is unused)
           - 6 -- erase a block, *arg* is the block number to erase
 
-See :ref:`filesystem` for example implementations of block devices using both
-protocols.
+Unless otherwise stated ``ioctl(op, arg)`` can return None. Consequently an
+implementation can ignore unused values of ``op``. As a minimum value 4 must be
```

Comment:
"As a minimum value 4" -> "As a minimum, value 4" (or As a minimum `ioctl(4,...)`", I was reading "minimum value" and that didn't make sense, took me while to understand that you mean "minimum implementation".